http-verbs-common/src/test/scala/uk/gov/hmrc/http/RetriesSpec.scala

```diff
@@ -337,7 +337,7 @@ class RetriesSpec extends AnyWordSpecLike with Matchers with MockitoSugar with S
 
       implicit val headerCarrier: HeaderCarrier = HeaderCarrier()
 
-      http.POSTForm[Option[String]](url = "doesnt-matter", Map()).futureValue shouldBe None
+      http.POSTForm[Option[String]](url = "doesnt-matter", Map.empty[String, Seq[String]], Seq.empty[(String, String)]).futureValue shouldBe None
```

Comment:
Without specifying the type `[String, Seq[String]]` on Map, the compiler isn't able to work out which of the overloaded `POSTForm` we want to use. This seems to only happen on `POSTForm`. Upgrading to this version of http-verbs can result in compiler errors so wanted to point this out.

Reply:
I guess passing an empty body to POSTForm is unusual. If we don't have a problem with empty query params, I don't think it's an issue.